lepton/package.go

```diff
@@ -295,10 +295,11 @@ func GetPackageManifestFile() string {
 func PackageManifestChanged(fino os.FileInfo, remoteURL string) bool {
 	res, err := http.Head(remoteURL)
 	if err != nil {
-		if err, ok := err.(net.Error); ok {
+		var netError *net.Error
+		if errors.Is(err, *netError) {
 			fmt.Printf(constants.WarningColor, "missing internet?, using local manifest.\n")
 		} else {
-			panic(err)
+			fmt.Printf("probably bad URL: %s, got error %s", remoteURL, err)
```

Comment:
since this was a panic need to exit here

Reply:
done